src/behandlinger/behandlingsresultattyper.js

```diff
@@ -35,6 +35,14 @@ const behandlingsresultattyper = [
   {
     kode: 'MEDLEM_I_FOLKETRYGDEN',
     term: 'Medlem i folketrygden'
+  },
+  {
+    kode: 'AVSLUTTET_UTEN_ENDRING',
+    term: 'Behandlingen er avsluttet'
```

Comment:
I https://confluence.adeo.no/display/TEESSI/Kodeverk+i+Melosys#KodeverkiMelosys-BehandlingsResultatTyper står det "Avslutt behandlingen" som term. Ser ikke ut til å henge sammen for et resultat.
Sjekk med fag

Reply:
Har vært litt fram og tilbake med dette. Har snakket med fag igjen, og vi bestemte oss for å endre det nok en gang. Både koden og Confluence-siden har blitt oppdatert.

Veldig nyttig at du alltid lenker til dokumentasjon som jeg ikke vet eksisterer. Takk!